application/frontend/src/pages/Feed.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from "react";
+import axios from "axios";
+import { List, Typography, Divider, Avatar, Card } from 'antd';
+import { Link } from "react-router-dom";
+
+export function Feed() {
+
+    const user_id = localStorage.getItem("user_id")
+    const token = localStorage.getItem("token")
+
+
+    type FeedElement = {
+        actor: {
+            type: string,
+            name: string
+        },
+        createdAt: string,
+        object: string | null,
+        objectschema: {
+            content: string,
+            id: number,
+            name: string,
+            type: string,
+            url: string
+        } | null,
+        origin: {
+            id: number,
+            name: string,
+            type: string
+        } | null,
+        summary: string,
+        target: {
+            content: string,
+            id: 0,
+            name: string,
+            type: string,
+            url: string
+        } | null,
+        type: string
+    }
+
+    type FeedElement_short = {
+        summary: string,
+        actor_name: string,
+        url: string,
+        target_id: number,
+        target_name:string,
+        time:string
+    }
+
+    const initialFeedData: FeedElement_short[] = []
+
+    const [feedData, setFeedData] = React.useState(initialFeedData);
+    const [summaries, setSummary] = React.useState([""])
+    const [actor_names, setActor_Names] = React.useState([""])
+    const [urls, setUrls] = React.useState([""])
+    //const [feedData, setFeedData] = React.useState([])
+
+    React.useEffect(() => {
+        axios
+            .get(`/activitystreams/${user_id}`, {
+                headers: { Authorization: `Bearer ${token}` },
+                data: {},
+            })
+            .then((response) => {
+                // check for error response
+                if (response.status === 200) {
+                    let data = response.data
+
+                    let tmp_Feed_Data = []
+
+                    for (let i = 0; i < data.length; i++) {
+                        var tmp_Feed_Datum = { summary: "", actor_name: "", url: "", target_id: -1 ,target_name: "",time:""}
+                        tmp_Feed_Datum.summary = data[i].summary
+                        tmp_Feed_Datum.actor_name = data[i].actor.name
+                        tmp_Feed_Datum.url = data[i].objectschema.url
```

Comment:
Could you please change the Route for the profile pages in App.tsx to match this URL? Here is the current route:
<Route path="/profile/:target_id" children={<ProfilePage /> } />

Reply:
I'm on it

Reply:
[issue #535](https://github.com/bounswe/2021SpringGroup12/issues/535) is created for it. ıt will be easier after the merge